packages/tools/src/synchronizers/callbacks/cameraSyncCallback.ts

```diff
@@ -34,9 +37,15 @@ export default function cameraSyncCallback(
   const tViewport = renderingEngine.getViewport(targetViewport.viewportId);
 
   // TODO: only sync in-plane movements if one viewport is a stack viewport
+  if (camera.parallelProjection) {
+    const sViewport = renderingEngine.getViewport(sourceViewport.viewportId);
+    const srcDelta = sViewport.getPan();
 
-  // Todo: we shouldn't set camera, we should set the focalPoint
-  // to the nearest slice center world position
-  tViewport.setCamera(camera);
+    // Do the zoom first, as the pan is relative to the zoom level
+    tViewport.setZoom(sViewport.getZoom());
```

Comment:
I don't like the inconsistensy in the set/get here. Can we have `const srcZoom = ...` similar to the `const srcDelta ...` instead of the `.setZoom(sViewport.getZoom())` ?